synapse/lib/ast.py

```diff
@@ -24,6 +26,40 @@ def __init__(self, item=None):
 class StormBreak(StormCtrlFlow): pass
 class StormContinue(StormCtrlFlow): pass
 
+@s_cache.memoize()
+def getGlobRegex(text):
```

Comment:
Could we move this to somewhere more generic, like synapse/lib/cache.py (where the other glob is)

Reply:
(cuz I'm gonna have to use it for heat)

Reply:
Actually, we now have two implementations. Could you see if you can merge the two?